.github/workflows/vol_rest.yml

```diff
@@ -191,5 +191,5 @@ jobs:
           HSDS_ENDPOINT=http+unix://%2Ftmp%2Fhs%2Fsn_1.sock \
           ROOT_DIR=${{github.workspace}}/hsdsdata \
           BUCKET_NAME=hsdstest \
-          ctest --build-config ${{ inputs.build_mode }} -VV -R "h5_api" -E "H5COPY|H5DIFF" .
+          ctest --build-config ${{ inputs.build_mode }} -VV -R "h5_api" -E "H5COPY|H5DIFF|H5LS" .
```

Comment:
Do we have a sense of why the REST VOL can't be run with these tools tests? It supports a good portion of the HDF5 API and at some point we should be running the tools tests with all the VOLs we support.